snapcraft/plugins/python2.py

```diff
@@ -137,23 +145,11 @@ def build(self):
         os.makedirs(self.dist_packages_dir, exist_ok=True)
```

Comment:
Weird that it would need to be created; no I see our unit test coverage sucks for these python plugins :sob: